cheatsheets/Transport_Layer_Security_Cheat_Sheet.md

```diff
@@ -136,19 +136,17 @@ For internal applications, an internal CA can be used. This means that the FQDN
 
 Certification Authority Authorization (CAA) DNS records can be used to define which CAs are permitted to issue certificates for a domain. The records contains a list of CAs, and any CA who is not included in that list should refuse to issue a certificate for the domain. This can help to prevent an attacker from obtaining unauthorized certificates for a domain through a less-reputable CA. Where it is applied to all subdomains, it can also be useful from an administrative perspective by limiting which CAs administrators or developers are able to use, and by preventing them from obtaining unauthorized wildcard certificates.
 
-### Always Provide All Needed Certificates
+### Consider the Certificate’s Validation Type
 
-In order to validate the authenticity of a certificate, the user's browser must examine the certificate that was used to sign it and compare it to the list of CAs trusted by their system. In many cases the certificate is not directly signed by a root CA, but is instead signed by an intermediate CA, which is in turn signed by the root CA.
+Certificates come in types of validation. Validation is the process the Certificate Authority uses to make sure you are allowed to have the certificate. This is authorization. The base validation is called Domain Validated (DV). All publicly issued certificates must be domain validated. This process involves practical proof of control of the name or endpoint requested in the certificate. This usually involves a challenge and response in DNS, to an official email address, or to the endpoint that will get the certificate.
 
-If the user does not know or trust this intermediate CA then the certificate validation will fail, even if the user trusts the ultimate root CA, as they cannot establish a chain of trust between the certificate and the root. In order to avoid this, any intermediate certificates should be provided alongside the main certificate.
+Organization Validated (OV) certificates include the subscriber’s organization information in the certificates subject DN. E.g. C = GB, ST = Manchester, **O = Sectigo Limited**, CN = sectigo.com. The process to acquire an OV certificate requires official contact with the subscriber company via a method that proves to the CA that they are truly talking to the right company.
```

Comment:
additionally include?

I'd change subscriber to certificate subject as the point (for the certificate using/relying audience) is the certificate, not payment for it.

Reply:
subscriber is the official term of the requestor for paid and free certs. Subject is the name that goes in the cert subject DN. I will kill that subscriber in favor of requestor .